lib/hanami/cli/generators/gem/app/puma.erb

```diff
@@ -6,10 +6,12 @@ threads min_threads_count, max_threads_count
 
 port        ENV.fetch("<%= Hanami::Port::ENV_VAR %>", <%= Hanami::Port::DEFAULT %>)
 environment ENV.fetch("HANAMI_ENV", "development")
-workers     ENV.fetch("HANAMI_WEB_CONCURRENCY", 2)
+workers     ENV.fetch("HANAMI_WEB_CONCURRENCY", 0)
 
-on_worker_boot do
-  Hanami.shutdown
+if ENV.fetch("HANAMI_WEB_CONCURRENCY", 0) > 0
+  on_worker_boot do
```

Comment:
@parndt Why this change? `Hanami.shutdown` is meant to free resources like database connection.

Reply:
@jodosha basically because it printed a warning telling me to: 
>Warning: You specified code to run in a `on_worker_boot` block, but Puma is not configured to run in cluster mode (worker count > 0 ), so your `on_worker_boot` block did not run
>Puma starting in single mode...